test/build/50_ch-image.bats

```diff
@@ -329,3 +329,13 @@ EOF
 }
 EOF
 }
+
+@test 'ch-image push: consitent layer hash' {
+ run ch-image push --hash-only 00_tiny
+ [[ $status -eq 0 ]]
+ echo "$output"
+ [[ $output = *'gathering'*'preparing'*'metadata'*'layer 1/1'* ]]
+ out=$(ch-image push --hash-only 00_tiny)
+ out2=$(ch-image push --hash-only 00_tiny)
+ [[ $out = "$out2" ]]
```

Comment:
Are we confident the output is completely identical, not just the hashes? If so, that choice needs a comment IMO.